test_suite/dmqmc/np2/h_cmca4_ks_complex/c_ipdmqmc.in

```diff
@@ -6,9 +6,9 @@ system = read_in {
 dmqmc {
 	sys = system,
 	qmc = {
-		tau = 0.01,
+		tau = 0.001,
 		rng_seed = 200,
-		init_pop = 200,
+		init_pop = 2000,
```

Comment:
Requires a new benchmark.

Reply:
I did this on purpose. Why new benchmark?

Reply:
So the original test had too large a time step and normally I would set the initial population to be close to the desired population when running at high temperatures. I also increased the temperature to shorten the test length. But, the input file matches the benchmark so I don't  think it needs to be redone.

Reply:
Sorry, I missed the commit where you also redid the benchmarks! My bad...